requirements.txt

```diff
@@ -9,4 +9,5 @@ lockfile>=0.12.2
 pycrypto>=2.6.1
 retrying>=1.3.3
 parallel-ssh==1.9.1
+ssh2-python==0.19.0
```

Comment:
On this line, and in setup.py, some `# HACK – … ` comment describing when the pinning can be removed would be nice. Otherwise no-one will remove it.
e.g. a link to an upstream fix, or just referencing that 0.20.0 is broken and trying unpinning it when >0.20.0 is out